cluster/terraform_kubernetes/config/production.tfvars.json

```diff
@@ -92,6 +92,12 @@
    },
    "tra-production/check-childrens-barred-list-production-worker": {
       "receiver":"SLACK_WEBHOOK_CCBL"
-   }
+   },
+    "git-production/get-into-teaching-app-production": {
```

Comment:
The indentation is a bit off. Can you fix the indentation for this and the others in this file?

Reply:
done